src/main/groovy/org/grails/pages/Page.groovy

```diff
@@ -182,7 +182,7 @@ abstract class Page implements HtmlPage {
                 div(class: 'ocihometograils') {
                     span 'Sponsored by'
                     a(href: "https://objectcomputing.com/products/grails/") {
-                        img(src: 'assets/oci_home_to_grails.svg', alt: 'Object Computing - Home to Grails', width: '300px')
+                        img(src: '/assets/oci_home_to_grails.svg', alt: 'Object Computing - Home to Grails', width: '300px')
```

Comment:
@puneetbehl Is this going to generate a link to https://objectcomputing.com/products/grails/assets/oci_home_to_grails.svg?

Reply:
@jeffbrown It should point here: https://plugins.grails.org/assets/oci_home_to_grails.svg

Reply:
Roger that.  Thanks.  If `https://objectcomputing.com/products/grails/` above were changed to `https://objectcomputing.com/something/else/` would the same thing happen?  I am trying to understand the significance of the trailing /product/grails/` in the `href`.  Thanks again!

Reply:
Nevermind... the href is for the anchor, not the image.  My mistake.  Thanks!